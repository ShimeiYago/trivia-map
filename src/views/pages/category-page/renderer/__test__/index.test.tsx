import { shallow, ShallowWrapper } from 'enzyme';
import { Renderer, Props } from '..';

let wrapper: ShallowWrapper<Props, null, Renderer>;

const basicProps: Props = {
  categoryId: 1,
};

describe('Shallow Snapshot Tests', () => {
  beforeEach(() => {
    wrapper = shallow(<Renderer {...basicProps} />);
  });

  it('basic', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
