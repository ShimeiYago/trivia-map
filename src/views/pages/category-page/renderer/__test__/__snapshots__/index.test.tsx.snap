// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests basic 1`] = `
<HeadAppender
  title="隠れミッキー - トリビアマップ"
>
  <ArticleWrapper
    showSidebar={true}
  >
    <ForwardRef(Typography)
      align="center"
      component="h2"
      sx={
        Object {
          "mb": 2,
        }
      }
      variant="h6"
    >
      <IconAndText
        iconComponent={
          <Memo
            fontSize="inherit"
          />
        }
        iconPosition="left"
        text="隠れミッキー"
        variant="inherit"
      />
    </ForwardRef(Typography)>
    <ArticlePreviewList
      searchConditions={
        Object {
          "category": 1,
        }
      }
      variant="large"
    />
  </ArticleWrapper>
   
</HeadAppender>
`;
